.github/CODEOWNERS

```
@@ -564,6 +564,7 @@ src/platform/packages/shared/kbn-rule-data-utils @elastic/security-detections-re
 src/platform/packages/shared/kbn-safer-lodash-set @elastic/kibana-security
 src/platform/packages/shared/kbn-saved-search-component @elastic/obs-ux-logs-team
 src/platform/packages/shared/kbn-scout @elastic/appex-qa
+src/platform/packages/shared/kbn-scout-upgrade-testing @elastic/appex-qa
```

```suggestion
src/platform/packages/shared/kbn-scout-release-testing @elastic/appex-qa
```
...and update the PR description as well please.